index.html

```diff
@@ -7,20 +7,230 @@
+                <button class="find-food" type="submit">Find Food</button>
+              </div>
+              <p class="small-text">
+                <a href="" class="sign-in-sm">sign in</a>
+                for recent address
+              </p>
+            </form>
+            <p class="chopP">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+              Architecto modi facere rerum sequi veritatis iusto enim.
+            </p>
+          </div>
+        </div>
+      </section>
+      <!-- dishes section -->
+      <section id="dish-section">
+        <div class="card-header">
+          <h2 class="main-dishes">Main Dishes</h2>
+        </div>
+
+        <ul class="dishes">
+          <li class="dish-card">
+            <img
+              src="./Assets/other-images/Jollof-rice.jpeg"
+              alt=""
+              class="dish-img"
+            />
+            <h3 class="dish-name">Italian Pasta</h3>
+            <p class="dish-description">Mashed potatoes with T-bone steak...</p>
+          </li>
+
+          <li class="dish-card">
+            <img
+              src="./Assets/other-images/Jollof-rice.jpeg"
+              alt=""
+              class="dish-img"
+            />
+            <h3 class="dish-name">Italian Pasta</h3>
+            <p class="dish-description">Mashed potatoes with T-bone steak...</p>
+          </li>
+
+          <li class="dish-card">
+            <img
+              src="./Assets/other-images/Jollof-rice.jpeg"
+              alt=""
+              class="dish-img"
+            />
+            <h3 class="dish-name">Italian Pasta</h3>
+            <p class="dish-description">Mashed potatoes with T-bone steak...</p>
+          </li>
+
+          <li class="dish-card">
+            <img
+              src="./Assets/other-images/Jollof-rice.jpeg"
+              alt=""
+              class="dish-img"
+            />
+            <h3 class="dish-name">Italian Pasta</h3>
+            <p class="dish-discription">Mashed potatoes with T-bone steak...</p>
+          </li>
+
+          <li class="dish-card">
+            <img
+              src="./Assets/other-images/Jollof-rice.jpeg"
+              alt=""
+              class="dish-img"
+            />
+            <h3 class="dish-name">Italian Pasta</h3>
+            <p class="dish-discription">Mashed potatoes with T-bone steak...</p>
+          </li>
+        </ul>
+
+        <div class="see-more">
+          <a href="#" class="see"> See More Dishes</a>
+        </div>
+      </section>
+      <!-- reviews section -->
+      <section id="reviews">
+        <h2 class="review-header">What our users have to say!!</h2>
+
+        <div class="reviews-container">
+          <!-- content is generated dynamically with javaScript -->
+        </div>
+
+        <div class="see-more">
+          <a href="#" class="black-see"> See More </a>
+        </div>
+      </section>
+      <!-- partners section -->
+      <section id="partners">
```

Comment:
- [ ]  Kindly add alt tags to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible, add all alt tags in this section 👍🏿